src/pretix/static/pretixpresale/js/widget/widget.js

```diff
@@ -1986,8 +2000,19 @@ var create_button = function (element) {
         }
     }
 
+    var observer = new MutationObserver((mutationList) => {
+        mutationList.forEach((mutation) => {
+            if (mutation.type == "attributes" && mutation.attributeName.startsWith("data-")) {
+                Vue.set(app.widget_data, mutation.attributeName, mutation.target.getAttribute(mutation.attributeName));
```

Comment:
Don't we need to strip the `data-` here? At least in create_widget, we do it:

```
    for (var i = 0; i < element.attributes.length; i++) {
        var attrib = element.attributes[i];
        if (attrib.name.match(/^data-.*$/)) {
            widget_data[attrib.name.replace(/^data-/, '')] = attrib.value;
        }
    }
```

Reply:
Good catch!